src/3_PB/PianoDiProgetto/sections/ConsuntivoSprint/NonoSprint.typ

```diff
@@ -13,8 +13,8 @@
-Una strategia efficace per inibire il rischio RC1 è stata la scelta di svolgere gli #glossary("Stand-up meeting") remoti con una frequenza di due volte a settimana, che in unione al meeting interno settimanale, ha permesso di tenere sotto controllo l'avanzamento delle attività e di discutere tempestivamente eventuali problematiche, risultando un impegno sostenibile per il gruppo e quindi una pratica da mantenere anche in futuro.
+La retrospettiva ha evidenziato come il preventivo del totale delle ore sia stato rispettato. Tuttavia possiamo notare alcune differenze nell'ammontare delle ore di alcuni ruoli. In particolare, il ruolo del Verificatore è quello che risulta più differente rispetto al preventivo, con una totale delle ore inferiore. Questo non ci risulta di particolare preoccupazione. Infatti, si motiva con il fatto che le modifiche e integrazioni apportate ai documenti sono state minori rispetto a #glossary[sprint] passati e avendo due Verificatori la verifica è stata piuttosto rapida. Notiamo invece l'accuratezza del preventivo per il ruolo di Progettista, che nonostante venga utilizzato per la prima volta, con l'inizio della stesura delle _Specifiche Tecniche v1.0_, risulta esser stato stimato con precisione, indice di una maturata consapevolezza nella pianificazione delle attività. Con un leggero superamento le ore da Analista preventivate hanno permesso di rispettare la pianificazione e di completare la correzione dell'_Analisi dei Requisiti v1.0_ secondo le correzioni ricevute in revisione #glossary[RTB].
+Con la fine delle lezioni il team non ha più avuto modo di incontrarsi fisicamente; questo ha suscitato la preoccupazione di un potenziale rischio RC1. Una strategia che si è rivelata efficace nell'inibire questo rischio è stata la scelta di svolgere gli #glossary("Stand-up meeting") remoti con una frequenza di due volte a settimana, che in unione al meeting interno settimanale, ha permesso di tenere sotto controllo l'avanzamento delle attività e di discutere tempestivamente eventuali problematiche, risultando un impegno sostenibile per il gruppo e quindi una pratica da mantenere anche in futuro.
 
 ==== Aggiornamento della pianificazione e gestione dei rischi
 Il gruppo ha preso coscienza del fatto che il preventivo per questo #glossary[sprint] considerava fin troppe poche ore rispetto a quelle che si potevano impiegare a livello di risorse e che sono necessarie a completare le attività pianificate per la revisione #glossary[PB]\; di conseguenza, in ottica di arginare il rischio RP3, vengono aggiornati i preventivi successivi in modo da aumentare le ore di lavoro. In particolare, poiché questi primi #glossary[sprint] della #glossary[PB] richiedono principalmente attività di progettazione, si prevede un aumento delle ore per il ruolo di Progettista, ruolo che sarà ricoperto da un maggior numero di membri del gruppo.
```

Comment:
```suggestion
Il gruppo ha preso coscienza del fatto che il preventivo per questo #glossary[sprint] considerava fin troppe poche ore rispetto a quelle che si potevano impiegare a livello di risorse e che sono necessarie a completare le attività pianificate per la revisione #glossary[PB]\; di conseguenza, nell'ottica di arginare il rischio RP3, vengono aggiornati i preventivi successivi in modo da aumentare le ore di lavoro. In particolare, poiché questi primi #glossary[sprint] della #glossary[PB] richiedono principalmente attività di progettazione, si prevede un aumento delle ore per il ruolo di Progettista, ruolo che sarà ricoperto dunque svolto  da più  membri del gruppo contemporaneamente.
```

Reply:
alla fine aggiungerei una roba del tipo "la decisione viene inoltre motivata dal fatto che per l'attività di progettazione si vuole permettere la discussione tra più membri in modo tale da poter trovare l'architettura più adatta" una roba del genere, vedi te, se non ti va chiudi diriettamente il commento

Reply:
L'ho un po' modificata e aggiunto l'ultima cosa